app/controllers/chargeback_controller.rb

```diff
@@ -288,28 +288,28 @@ def cb_assign_field_changed
 
   # Add a new tier at the end
   def cb_tier_add
-    detail_index = params[:detail_index]
+    index = params[:index]
```

Comment:
What if I send an "evil" index here? I think it should throw an exception or something.